### ISSUE TITLE
Is this proposal still being explored?

### ISSUE THREAD

**Issue by dontcallmedom** [WebAudio/web-audio-cg#11]
Is the Audio Device Client API still under consideration / development? There hasn't be much activity in the repo in the past couple of years.

(if it isn't, it would be nice to update the readme/explainer accordingly, possibly to archive the repo)

**Comment by hoch** [comment]
No. There was a formal rejection from some WG members. The proposal is not being pursued anymore. I'll update the explainer accordingly.

Archiving this repo might need a WG discussion. We still might want to use this repo for the community group activities.
